src/main/kotlin/lotto/model/WinningCondition.kt

```diff
@@ -0,0 +1,17 @@
+package lotto.model
+
+import lotto.model.number.WinningNumber
+import lotto.model.number.WinningNumbers
+
+data class WinningCondition(val winningNumbers: WinningNumbers, val bonusNumber: WinningNumber) {
```

Comment:
아래의 코드로 개선해 보면 어떨까요?

```suggestion
data class WinningCondition(val winningNumbers: LottoNumbers, val bonusNumber: LottoNumber) {
```